src/imitation/policies/exploration_wrapper.py

```diff
@@ -57,7 +57,7 @@ def __init__(
 
     def _random_policy(
         self,
-        obs: np.ndarray,
+        obs: Union[np.ndarray, types.DictObs],
```

Comment:
Might be worth defining `Union[np.ndarray, types.DictObs]` as a type alias given it occurs so much.